pom.xml

```diff
@@ -85,7 +85,7 @@
       <activemq-artemis-native-version>2.0.0</activemq-artemis-native-version>
       <karaf.version>4.3.3</karaf.version>
       <pax.exam.version>4.13.4</pax.exam.version>
-      <commons.config.version>2.9.0</commons.config.version>
+      <commons.config.version>2.8.0</commons.config.version>
```

Comment:
Think the rebase went awry, this is now downgrading rather than not touching the version.